### PR TITLE
New package: Inkdrop.Inkdrop version 5.8.1

### PR DESCRIPTION
**Pull request by CoolPlayLin**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154144)

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
Hi @craftzdog
I've made this Pull Request to add inkdrop into windows package manager! Maybe this pull request needs the approval from you, its author

File: manifests/i/Inkdrop/Inkdrop/5.8.1/Inkdrop.Inkdrop.locale.en-US.yaml
Comment (craftzdog) [inline manifests/i/Inkdrop/Inkdrop/5.8.1/Inkdrop.Inkdrop.locale.en-US.yaml:8]: could you  change it to `https://www.inkdrop.app/`?

**Review by craftzdog — CHANGES_REQUESTED** [review]
Thanks, @CoolPlayLin! I will approve it when `PublisherUrl` is updated

**Comment by CoolPlayLin** [comment]
@craftzdog I've applied your suggestion. I shall put portable inkdrop into `Inkdrop.Inkdrop.Portable`

**Comment by CoolPlayLin** [comment]
https://github.com/microsoft/winget-pkgs/labels/Validation-Domain should be wavied

**Review by craftzdog — APPROVED** [review]
LGTM

**Comment by CoolPlayLin** [comment]
- Portable version is in #154151

File: manifests/i/Inkdrop/Inkdrop/5.8.1/Inkdrop.Inkdrop.locale.en-US.yaml
Comment (Trenly) [inline manifests/i/Inkdrop/Inkdrop/5.8.1/Inkdrop.Inkdrop.locale.en-US.yaml:7]:
```suggestion
Publisher: Takuya Matsuyama
```
This should match what is written to control panel
image

File: manifests/i/Inkdrop/Inkdrop/5.8.1/Inkdrop.Inkdrop.installer.yaml
Comment (Trenly) [inline manifests/i/Inkdrop/Inkdrop/5.8.1/Inkdrop.Inkdrop.installer.yaml:11]:
```suggestion
  InstallerSha256: DCA28559B2D8151592980AA52957462526124AEC44411A205BE076ECC1F658A1
  ProductCode: inkdrop
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Comment by CoolPlayLin** [comment]
https://github.com/microsoft/winget-pkgs/labels/Validation-Domain should be dropped
image

**Comment by craftzdog** [comment]
Configured the proper domain for the binary distribution: `dist.inkdrop.app`

**Comment by stephengillie** [comment]
> [ Validation-Domain](https://github.com/microsoft/winget-pkgs/labels/Validation-Domain) should be dropped image

Why should that label be dropped? The `PackageUrl` doesn't lead to the `InstallerUrl`. The image doesn't hold any data that would support this point - the existence of installer links doesn't guarantee that the `PackageUrl` will lead to them.

It appears that the developer hasn't released this version yet. It's possible that this might be a counterfeit upgrade, put out by another team to trick users into installing it.

**Comment by CoolPlayLin** [comment]
Hi @stephengillie



> Configured the proper domain for the binary distribution: `dist.inkdrop.app`

Issue has been resolved

**Comment by stephengillie** [comment]
I'm having difficulty getting this package to install correctly. 
- Normal install - ends with a setup error message, despite package manager reporting install complete. Shortcuts don't do anything.
- Retry with `VCRedist` - ends in normal install. Clicking shortcuts gives "This shortcut target has been removed." 

This might be a nested package, as one of the only `EXE`s it adds to the filesystem is `Setup.exe`, and after running this, Inkdrop appears to be installed.

**Pull request closed without merging**
